<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="truyenltpc" failures="1" tests="1" name="tetsCaseTinhNangChung.TC06_KiemTraSoLuongTaskVaBoldTask" time="92.465" errors="0" timestamp="2021-08-13T15:11:02 ICT" skipped="0">
  <testcase classname="tetsCaseTinhNangChung.TC06_KiemTraSoLuongTaskVaBoldTask" name="TGDChiaSeVB" time="92.465">
    <failure message="Task đã hoàn tất did not expect [Chia sẻ văn bản đến &quot;Kiểm tra số lượng Task và Bold Task: 13-08-2021 15:09:29&quot;] but found [Chia sẻ văn bản đến &quot;Kiểm tra số lượng Task và Bold Task: 13-08-2021 15:09:29&quot;]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Task đã hoàn tất did not expect [Chia sẻ văn bản đến "Kiểm tra số lượng Task và Bold Task: 13-08-2021 15:09:29"] but found [Chia sẻ văn bản đến "Kiểm tra số lượng Task và Bold Task: 13-08-2021 15:09:29"]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failEquals(Assert.java:1041)
at org.testng.Assert.assertNotEqualsImpl(Assert.java:147)
at org.testng.Assert.assertNotEquals(Assert.java:1531)
at pages.DanhSachViecCanXuLy.ValidateHoanTatTaskChiaSe(DanhSachViecCanXuLy.java:52)
at tetsCaseTinhNangChung.TC06_KiemTraSoLuongTaskVaBoldTask.TGDChiaSeVB(TC06_KiemTraSoLuongTaskVaBoldTask.java:84)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TGDChiaSeVB -->
  <system-out>
https://daotaoeoffice.petrolimex.com.vn/plxtkdg/vanban/SitePages/VanBanDen/VanBanDenEdit.aspx?IDVB=5513&source=https://daotaoeoffice.petrolimex.com.vn/plxtkdg/vanban/default.aspx?page=1  </system-out>
</testsuite> <!-- tetsCaseTinhNangChung.TC06_KiemTraSoLuongTaskVaBoldTask -->
